shared/shared/flask_frontend/errors.py

```diff
@@ -69,6 +69,9 @@ def wrapper(*args, **kwargs):
                 "key": e.key,
                 "type_verbose": "MODEL_LOCKED",
             }
+        except Exception as e:
+            print(e, type(e))
```

Comment:
I'll leave it in since we sometimes get 500ers, and logging them would be good. Also (I think) on 500ers, we do not return a json response. We'll see, if this happens...